src/MatrixTools.js

```diff
@@ -26,30 +26,48 @@ module.exports = {
         return room.getCanonicalAlias() || room.getAliases()[0];
     },
 
-    isDirectMessageRoom: function(room, me, ConferenceHandler, hideConferenceChans) {
+    /**
+     * If the room contains only two members including the logged-in user,
+     * return the other one. Otherwise, return null.
+     */
+    getOnlyOtherMember(room, me) {
+        const joinedMembers = room.getJoinedMembers();
+
+        if (joinedMembers.length === 2) {
```

Comment:
hrmmm.... what will this do if you are invited to a room with two other members?